entities/src/main/java/org/odk/collect/entities/javarosa/filter/LocalEntitiesFilterStrategy.kt

```diff
@@ -0,0 +1,62 @@
+package org.odk.collect.entities.javarosa.filter
+
+import org.javarosa.core.model.CompareToNodeExpression
+import org.javarosa.core.model.condition.EvaluationContext
+import org.javarosa.core.model.condition.FilterStrategy
+import org.javarosa.core.model.instance.DataInstance
+import org.javarosa.core.model.instance.TreeReference
+import org.javarosa.xpath.expr.XPathEqExpr
+import org.javarosa.xpath.expr.XPathExpression
+import org.odk.collect.entities.javarosa.intance.LocalEntitiesInstanceAdapter
+import org.odk.collect.entities.storage.EntitiesRepository
+import java.util.function.Supplier
+
+class LocalEntitiesFilterStrategy(entitiesRepository: EntitiesRepository) :
```

Comment:
I think it would be good to have a comment for this class explaining its purpose. I know we have tests but I'm not sure it's enough to understand it quickly and it might be a problem in the future.